features/messages/impl/src/main/kotlin/io/element/android/features/messages/impl/timeline/components/reactionsummary/ReactionSummaryPresenter.kt

```diff
@@ -38,10 +37,6 @@ class ReactionSummaryPresenter @Inject constructor(
 ) : Presenter<ReactionSummaryState> {
     @Composable
     override fun present(): ReactionSummaryState {
-        LaunchedEffect(Unit) {
-            room.updateMembers()
```

Comment:
This `updateMembers` is called as soon as the presenter is used, and since this presenter is included in `MessagesPresenter`, this happens when the chat screen is displayed. The thing is, on `RoomLoadedFlowNode.fetchRoomMembers()` there is another call for this that will also get called when the chat screen is created, so there's a difference of a few ms between them, and in large rooms, this method is very expensive to run.

Reply:
I see, so the reason has nothing to do with this @Immutable refactor.

Reply:
Not really, sorry. I should have added a comment about this.